src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -463,9 +467,9 @@ function TotalValidWeeklySummaries({ summary, canEditSummaryCount }) {
   );
 }
 
-function Bio({ bioCanEdit, ...props }) {
+function Bio({ bioCanEdit, canRequestBio, ...props }) {
   // eslint-disable-next-line react/jsx-props-no-spreading
-  return bioCanEdit ? <BioSwitch {...props} /> : <BioLabel {...props} />;
+  return bioCanEdit || canRequestBio ? <BioSwitch {...props} /> : <BioLabel {...props} />;
```

Comment:
This should be replacing bioCanEdit instead of adding to it.

Reply:
And `bioCanEdit` (which is based on the previous `putUserProfileImportantInfo` I think) should be removed.